skimage/color/tests/test_colorconv.py

```diff
@@ -317,10 +317,19 @@ def test_xyz2lab(self):
                 assert_array_almost_equal(lab_array_I_obs,
                                           xyz2lab(self.xyz_array, I, obs),
                                           decimal=2)
-        for I in ["d75", "e"]:
-            fname = "color/tests/data/lab_array_{0}_2.npy".format(I)
-            lab_array_I_obs = np.load(fetch(fname))
-            assert_array_almost_equal(lab_array_I_obs,
+        for I in ["A", "B", "C", "d75"]:
+            I = I.lower()
+            for obs in ["2", "10"]:
+                obs = obs.lower()
+                fname = "color/tests/data/lab_array_{0}_{1}.npy".format(I, obs)
+                lab_array_I_obs = np.load(fetch(fname))
+                assert_array_almost_equal(lab_array_I_obs,
+                                          xyz2lab(self.xyz_array, I, obs),
+                                          decimal=2)
+        I= "e"
```

Comment:
Please add a space :slightly_smiling_face: 
```suggestion
        I = "e"
```